.yamato/com.unity.ml-agents-test.yml

```diff
@@ -35,6 +35,12 @@ test_{{ platform.name }}_{{ editor.version }}:
   dependencies:
     - .yamato/com.unity.ml-agents-pack.yml#pack
   triggers:
+    expression: |
+      pull_request.target eq "master" AND
+      NOT pull_request.draft AND 
+      (pull_request.changes.any match "com.unity.ml-agents/**" OR 
+      pull_request.changes.any match ".yamato/com.unity.ml-agents-test.yml")
+      
```

Comment:
Do we remove the `changes` section, or keep that in place?

Reply:
should remove it, was trying things out